backend/app/rest/api/rest_private.go

```diff
@@ -221,6 +223,39 @@ func (s *private) voteCtrl(w http.ResponseWriter, r *http.Request) {
 	render.JSON(w, r, R.JSON{"id": comment.ID, "score": comment.Score})
 }
 
+// TODO: Should be two-step process with email confirmation code/link,
```

Comment:
Note for me: figure out how to achieve that. Send verification email without token and set it via the email link, re-issuing the same request with it? Generate a token and verify email change via GET request? Should the token be time-bound?